Makefile

```diff
@@ -59,13 +59,13 @@ delete-account:
 test-account-creation: create-account delete-account
 
 # Create account claim namespace
-.PHONY: create-account-claim-namespace
+.PHONY: create-accountclaim-namespace
```

Comment:
Line 284 uses `create-account-claim-namespace` and need to be updated

Reply:
Fixed. Good catch.